src/styles/parts/footer.scss

```diff
@@ -0,0 +1,18 @@
+.footer {
```

Comment:
According to the checklist, footer should be sticky at the bottom with z-index: -1 to get a fixed background effect. I don't see any styles which provide this effect. Am I missing anything?